src/_companies/europe/turkiye/istanbul/shft/index.md

```diff
@@ -0,0 +1,26 @@
+---
+name: "SHFT"
+
+continent: "Europe"
+country: "Turkiye"
+region: "Istanbul"
+city: "Istanbul"
+
+logo: logo.png
+
+website: https://shft.co/
+careers_page: https://shft.co/career/
+
+work_arrangements:
+  - remote
+
+stack:
+  - rails
+  - postgres
+  - react
+  - mysql
+
+created_at: 2023-03-28 10:38:16
+---
+
+SHFT is a startup for the fast-growing tech companies. We take the development load off your shoulders with scalable and dedicated in-house developers. Think of us as your strategic product partner that codes. We welcome the enthusiastic startups and target-driven scale-ups; assist them with a thorough design, accompanied by a detailed MVP scope, followed by an impeccable development, and finally a confident product launch. We essentially cover your whole product building phase.
```

Comment:
Could you change this to be in the third person? It's the default for all other company descriptions :)

Reply:
I didn't realize it was that way. Changed it, thank you!